go.mod

```diff
@@ -0,0 +1,36 @@
+module gotify
```

Comment:
shouldn't this be `github.com/gotify/server`?

Reply:
I named the module gotify so that `go build` will automatically produce an executable called `gotify`(rather than the default `server`). I did observe some similar practices(naming the main module differenct from its import path) on other repos. I read some other repo's practices and found that is a rare case. I will have this changed. Thanks.